src/components/Column/__tests__/__snapshots__/Column.test.tsx.snap

```diff
@@ -571,7 +566,68 @@ exports[`Column should have correct style show column with correct style 1`] = `
                     </svg>
                   </button>
                 </div>
-              </div>
+              </header>
+              <main
+                class="note__text"
+              >
+                Lorem Ipsum
+              </main>
+              <footer
+                class="note__footer"
+              >
+                <div
+                  class="note-reaction-list__root"
+                >
+                  <div
+                    class="note-reaction-list__reaction-bar-container"
```

Comment:
can you make the reaction menu navigatable by keyboard? Right now, pressing tab when the menu is opened just closes it again. And pressing Enter when the toggle is focused opens the stack view.
Also, the menu toggle and the individual menu items need focus states

Reply:
it cost me a lot of nerves, but it should work now :)